src/components/PurchaseOrder/UpdateOrderErrorModal/UpdateOrderErrorModal.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Button, Modal } from '@folio/stripes/components';
+
+const UpdateOrderErrorModal = ({ orderNumber, cancel, reason }) => {
+  const footer = (
+    <Button
+      onClick={cancel}
+      marginBottom0
+    >
+      <FormattedMessage id="ui-orders.openOrderModal.cancel" />
+    </Button>
+  );
+
+  return (
+    <Modal
+      label={<FormattedMessage id="ui-orders.openOrderModal.title" values={{ orderNumber }} />}
```

Comment:
multiple props is better to break up on multiple lines

Reply:
rearranged